frontend/package-lock.json

```diff
@@ -7782,9 +7763,9 @@
       }
     },
     "lodash": {
-      "version": "4.17.14",
```

Comment:
we need to retain this version due to a vulnerability... be sure you rebase on master branch.